src/main/java/kitchenpos/menus/tobe/domain/vo/MenuName.java

```diff
@@ -0,0 +1,51 @@
+package kitchenpos.menus.tobe.domain.vo;
+
+import java.io.Serializable;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import kitchenpos.menus.tobe.domain.exception.EmptyMenuNameException;
+import kitchenpos.menus.tobe.domain.exception.ProfanityMenuNameException;
+import kitchenpos.products.infra.PurgomalumClient;
+
+@Embeddable
+public class MenuName implements Serializable {
+
+    @Column(name = "name", nullable = false)
+    private String value;
```

Comment:
```suggestion
    private Name name;
```
으로 변경한다면 이름이 가져야하는 기본적인 isBlank를 격리 시키고 각 컨텍스트별 이름의 요구사항만 포함 할 수 있을것 같아요.

Reply:
공감합니다!
이름을 포함한 가격, 수량 밸류도 공통 밸류 객체로 추출하겠습니다.